index.js

```diff
@@ -47,4 +47,11 @@ function createError (code, message, statusCode = 500, Base = Error) {
   return FastifyError
 }
 
+function isFastifyError (value) {
+  return Object.prototype.toString.call(value) === '[object FastifyError]'
```

Comment:
I'm hardly -1 on this approach.

Reply:
@mcollina @jsumners suggested this approach. What should we agree on?

Reply:
Would it work with a custom symbol?

Reply:
No, the `toStringTag` is a spec defined symbol.

Reply:
We can just set our custom
Symbol for every instance created by the constructor and check for it. We already do this in our codebase.

Reply:
True. But we are delving deeper into hack territory.